src/main/java/seedu/address/logic/parser/AddExamCommandParser.java

```diff
@@ -27,13 +28,13 @@ public AddExamCommand parse(String args) throws ParseException {
         if (!arePrefixesPresent(argMultimap, PREFIX_ID, PREFIX_DATE) || !argMultimap.getPreamble().isEmpty()) {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddExamCommand.MESSAGE_USAGE));
         }
-        assert argMultimap.getValue(PREFIX_ID).isPresent() : "Prefix ID value is missing";
-        assert argMultimap.getValue(PREFIX_EXAM_NAME).isPresent() : "Prefix Exam Name value is missing";
-        assert argMultimap.getValue(PREFIX_DATE).isPresent() : "Prefix Date value is missing";
 
         String uniqueId = argMultimap.getValue(PREFIX_ID).get();
         String examName = ParserUtil.parseExamName(argMultimap.getValue(PREFIX_EXAM_NAME).get());
         LocalDate date = ParserUtil.parseDate(argMultimap.getValue(PREFIX_DATE).get());
+        if (date.isBefore(LocalDate.now())) {
+            throw new ParseException(MESSAGE_CONSTRAINTS_PAST_DATE);
+        }
```

Comment:
am i able to still add exams on the same day but at a past timing?

Reply:
Yes, we are still able to add a exam date that is on the same day but is at a past timing, 

Adding exams past the current date will throw the correct error message though